config/docker-monitor.json

```diff
@@ -26,6 +31,12 @@
 			"condition": "CorruptDockerOverlay2",
 			"reason": "CorruptDockerOverlay2",
 			"pattern": "returned error: readlink /var/lib/docker/overlay2.*: invalid argument.*"
+		},
+		{
+			"type": "permanent",
```

Comment:
I remember you said this is a temporary issue.

Reply:
yes, but later I verfied with my team and figured out that it needs a change.